### PR TITLE
SCXA SolrCloud image

### PR DESCRIPTION
**Pull request by alfonsomunozpomer**
An image that contains an empty SolrCloud instance ready to use in testing and CI workflows. It’s just one SolrCloud node running with an embedded ZooKeeper.

I’ve added an action to publish to Quay but until it runs I won’t know for sure it works.

### CODE REVIEW

File: .github/workflows/scxa-solrcloud.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    if: github.ref == 'refs/heads/master'
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Read upstream tag without version
+      id: gettag
+      run: echo "::set-output name=tag::$(head -n 1 scxa-solrcloud/Dockerfile | awk -F':' '{print $2}' | awk -F'-' 'BEGIN { OFS="-" } {$NF=""; print $0}')"
+
+    - name: Read internal update version
```
Comment (pinin4fjords) [inline .github/workflows/scxa-solrcloud.yaml:23]: You don't have the RUNNER_VERSION issue the other container has, so you should remove this step...

File: .github/workflows/scxa-solrcloud.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      id: getversion
+      run: echo "::set-output name=version::$(grep 'ARG RUNNER_VERSION' scxa-solrcloud/Dockerfile | awk -F'=' '{print $2}')"
+
+    - name: Build image
+      id: build-image
+      uses: redhat-actions/buildah-build@v2
+      with:
+        image: ${{ env.IMAGE_NAME }}
+        tags: latest ${{ steps.gettag.outputs.tag }}${{ steps.getversion.outputs.version }}
```
Comment (pinin4fjords) [inline .github/workflows/scxa-solrcloud.yaml:32]: .. and remove the getversion reference here.

```suggestion
        tags: latest ${{ steps.gettag.outputs.tag }}
```

**Review by pinin4fjords — CHANGES_REQUESTED** [review]

**Comment by alfonsomunozpomer** [comment]
Thanks @pinin4fjords for the suggestions. I just copied the GH Actions Runner YAML just in case there were some particularities of GH actions I wasn’t aware of. I modified the action as per your suggestions. Let me know how that looks now.

**Review by pinin4fjords — APPROVED** [review]

**Comment by pinin4fjords** [comment]
Looks good!

**Comment by pinin4fjords** [comment]
Though @alfonsomunozpomer actually, we might want chat with @pcm32 about the submodule pattern. It might be better to use the git checkout action instead to clone the additional repos https://github.com/actions/checkout#checkout-multiple-repos-nested

**Comment by alfonsomunozpomer** [comment]
Ah, the image build failed because of submodules not being in the expected path. I’ve created and assigned https://github.com/ebi-gene-expression-group/atlas-containers/issues/6.

**Pull request merged**